docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-discord.mdx

```diff
@@ -277,7 +277,7 @@ If everything works fine, the log output should look like this:
 
 ```code
 $ teleport-discord start
-INFO   Starting Teleport Access Discord Plugin (=teleport.version=): discord/app.go:80
+INFO   Starting Teleport Zero Trust Access Discord Plugin (=teleport.version=): discord/app.go:80
```

Comment:
Similar to previous comments, we should only change this if we change the code, otherwise the docs don't reflect reality.